.github/workflows/publish_docker_image.yaml

```diff
@@ -0,0 +1,85 @@
+name: Docker
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Our "main" branch is still called `master`.
Do we want to start using `main` instead of `master`?

Reply:
Renamed this to `master`